content/options.js

```diff
@@ -20,6 +20,34 @@ const Options = {
 
     let options = document.getElementById("options");
     options.addEventListener("change", this);
+
+    this.initWorkingHours();
+  },
+
+  async initWorkingHours() {
+    // Specify reasonable defaults for the first-run case.
+    let { workingHours } = await browser.storage.local.get({workingHours: {
+      enabled: false,
+      startTime: "09:00",
+      endTime: "17:00",
+      days: ["monday","tuesday","wednesday","thursday","friday"]
+    }});
+
+    document.querySelector("#working-hours-checkbox").checked = workingHours.enabled;
+    if (workingHours.enabled) {
+      document.querySelector("#working-hours").removeAttribute("disabled");
+    } else {
+      document.querySelector("#working-hours").setAttribute("disabled", "disabled");
+    }
+    document.querySelector("#start-time").value  = workingHours.startTime;
+    document.querySelector("#end-time").value    = workingHours.endTime;
+    document.querySelector("#sunday").checked    = workingHours.days.includes("sunday");
```

Comment:
I think I might prefer if we query the document for the days, and then loop over the nodes, setting the checked value if the IDs (or maybe better, if a data-day attribute) is included in workingHours.days.